src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -120,6 +120,10 @@ public static String sanitizeForHtml(String str){
                 .replaceAll("&(?!(amp;)|(lt;)|(gt;)|(quot;)|(#x2f;)|(#39;))", "&amp;");
     }
     
+    public static String sanitizeHtmlForSaving(String html) {
```

Comment:
This method is pretty odd - it just calls sanitizeForHtml, is this being used often?